src/components/FieldValueDirectAnswer.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  FieldValueDirectAnswer as FieldValueDirectAnswerType,
+  BuiltInFieldType,
+  Address
+} from '@yext/search-headless-react';
+import { useMemo } from 'react';
+
+/**
+ * Props for FieldValueDirectAnswer.
+ */
+interface FieldValueDirectAnswerProps {
+  /** A field value direct answer result. */
+  result: FieldValueDirectAnswerType,
+  /** Handle onClick event for "View Details" link. */
+  viewDetailsClickHandler?: () => void,
+  /** CSS classes for customizing the component styling. */
+  cssClasses?: FieldValueDirectAnswerCssClasses
+}
+
+/**
+ *  The CSS class interface for FieldValueDirectAnswer.
+ */
+interface FieldValueDirectAnswerCssClasses {
+  header?: string,
+  body?: string,
+  content?: string,
+  answerContainer?: string
+}
+
+/**
+ * Renders a field value direct answer provided by the Search API.
+ */
+export function FieldValueDirectAnswer({
+  result,
+  viewDetailsClickHandler,
+  cssClasses = {}
+}: FieldValueDirectAnswerProps): JSX.Element {
+  const title = `${result.entityName} / ${result.fieldName}`;
+  const link = result.relatedResult.link;
```

Comment:
out of curiosity, do you know why we only check this link to determine if the DA has a link? I was looking in the HH theme code and it seems like we use this link if it's present, and if not, we look for a `landingPageUrl` in the `rawData` and use that if we find it

Reply:
I think we might have based it off component.js in allfields-standard of the theme, which uses relatedResult.link (aka data.website). I see that `landingPageUrl` is only use documentsearch-standard of the theme for DA. I can update to fall back on landingPageUrl!